hooks/keymap/keymap

```diff
@@ -0,0 +1,16 @@
+# vim: set ft=sh:
+# shellcheck shell=sh
+#
+# false positive
+# shellcheck disable=2154
+#
+# handle_keymap()
+{
+    print "configuring keymap"
+
+    keymap_folder=$(dirname "$keymap_path")
+
+    mkdir -p "${tmpdir}${keymap_folder}"
+    cp "$keymap_path" "${tmpdir}${keymap_folder}"
```

Comment:
copy_file "$keymap_path" "${keymap_path%/*}" 644 0

Reply:
Changed that